tasks/check-vars.yml

```
@@ -0,0 +1,148 @@
+---
+- name: Print network configuration
+  debug:
+    msg: |
+      bbb_hostname:      {{ bbb_hostname }}
+      bbb_local_ip:      {{ bbb_local_ip }}
+      bbb_bind_ip4:      {{ bbb_bind_ip4 }}
+      bbb_public_ip4:    {{ bbb_public_ip4 }}
+      bbb_bind_ip6:      {{ bbb_bind_ip6 or 'disabled' }}
+      bbb_public_ip6:    {{ bbb_public_ip6 or 'disabled' }}
+      bbb_net_mtu:       {{ bbb_net_mtu }}
+      bbb_ufw_enable:    {{ bbb_ufw_enable }}
+      {% if bbb_ufw_enable %}
+      bbb_ufw_rules.ssh: {{ (bbb_ufw_rules_default | combine(bbb_ufw_rules)).ssh | to_yaml | trim }}
+      {% endif %}
+
+- name: Check minimum requirements
+  ansible.builtin.assert:
+    quiet: true
+    that: "{{ item }}"
+  loop:
+    - ansible_distribution_release | lower == bbb_ubuntu_name
+    - bbb_version.startswith(bbb_ubuntu_name)
+
+- name: Check core variables
+  when:
+    - bbb_github_ci is undefined
+  ansible.builtin.assert:
+    quiet: true
+    that: "{{ item }}"
+  loop:
+    - bbb_secret is defined
+    - bbb_hostname is fqdn
+    - bbb_apt_mirror is url
+    - bbb_apt_key is url
+    - bbb_disabled_features is subset(bbb_allowed_disabled_features)
+    # Security
+    - bbb_secret | length > 16
+    - bbb_secret_seed | length > 16
+    - bbb_host_seed | length > 16
+    - bbb_freeswitch_socket_password | length > 16
+    - bbb_freeswitch_default_password | length > 16
+    - bbb_coturn_secret | length > 16
+
+- name: Check IPv4 variables
+  when:
+    - bbb_bind_ip4 or bbb_public_ip4
+    - bbb_github_ci is undefined
+  ansible.builtin.assert:
+    quiet: true
+    that: "{{ item }}"
+  loop:
+    - bbb_local_ip is ip(loopback=True)
+    - bbb_bind_ip4 == bbb_public_ip4 or bbb_bind_ip4 is ip(v=4, private=True)
+    - bbb_public_ip4 is ip(v=4, public=True)
+
+- name: Check IPv6 variables
+  when:
+    - bbb_bind_ip6 or bbb_public_ip6
+    - bbb_github_ci is undefined
+  ansible.builtin.assert:
+    quiet: true
+    that: "{{ item }}"
+  loop:
+    - bbb_bind_ip6 in == bbb_public_ip6 or bbb_bind_ip6 is ip(v=6, private=True)
```

```suggestion
    - bbb_bind_ip6 == bbb_public_ip6 or bbb_bind_ip6 is ip(v=6, private=True)
```

Just tried the current revision, works great beside this typo 😄